README.md

```diff
@@ -1,14 +1,615 @@
 # Ably Chat SDK for Android
 
-This is the repository for the Android version of the Ably Chat SDK. We aim to build the same functionality that’s available in the [JavaScript SDK](https://github.com/ably/ably-chat-js).
+<p style="text-align: left">
+    <img src="https://img.shields.io/badge/development_status-Private_Beta-ab7df8" alt="Development status"   />
+    <img src="https://badgen.net/github/license/3scale/saas-operator" alt="License" />
```

Comment:
Can we also add one more badge for the current version, like
`ably-chat-kotlin:0.0.1-beta`

Reply:
done